### PR TITLE
[node] move to 13.9

### PR DESCRIPTION
**Pull request by Flarna**
Added semver minor changes from 13.8/13.9 except crypto.

Crypto seems already outdated and it is quite complicated. As I don't use these APIs it's too error prone that I add them. Would be nice if some crypto expert could do this.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V13.md#13.9.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Review by SimonSchick — CHANGES_REQUESTED** [review]
Blocking this for now, there's a bunch of things missing.
Also missing tests.

Will link to my WIP branch so you can just grab those changes in a bit.

**Comment by SimonSchick** [comment]
@Flarna https://github.com/DefinitelyTyped/DefinitelyTyped/commit/c4f9bc51e300b7fef72ac79a4af4d69807e38148 some of which you've already done.

**Review by amcasey — CHANGES_REQUESTED** [review]
Seems like @SimonSchick has questions and the bot isn't picking up their status.

File: types/node/globals.d.ts
Comment (ExE-Boss) [inline types/node/globals.d.ts]: This probably shouldn’t be removed.

**Review by ExE-Boss — CHANGES_REQUESTED** [review]

File: types/node/globals.d.ts
Comment (Flarna) [reply types/node/globals.d.ts]: why? What is the benefit of this?

Comment (SimonSchick) [reply types/node/globals.d.ts]: It's basically the same content as the actual EventEmitter definitions and has no value to the reader, we had it in a couple other places and it got removed over time.

I'd keep the `EventEmitter` line tho.

**Comment by Flarna** [comment]
Added tests and `worker.getHeapSnapshot()` (see https://github.com/nodejs/node/pull/32061 why not `takeHeapSnapshot()`).

As mentioned above I will not add crypto changes in this PR to avoid it gets to big.
This should be done via some followup PR.

File: types/node/globals.d.ts
Comment (Flarna) [reply types/node/globals.d.ts]: added back `EventEmitter`.

**Review by SimonSchick — APPROVED** [review]
@Flarna looks good!
Also 13.10 just relased, gotta go fast!

**Comment by SimonSchick** [comment]
@amcasey pls unblock ❤️

**Review by amcasey — APPROVED** [review]

**Pull request merged**
